utils/allocator/allocator.go

```diff
@@ -34,5 +35,24 @@ func (p *allocator) Release(item string) {
 	if len(p.allocate) == cap(p.allocate) {
 		return
 	}
+
+	intItem, err := strconv.Atoi(item)
+	if err == nil {
```

Comment:
looks like the check is inverted? Also we should check if intItem is 0 and ignore releasing it like we do for ReleaseInt.